.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
```

Comment:
Oh I haven't seen that yet, do you know what it does?

Reply:
I just saw [these docs](https://help.github.com/en/github/administering-a-repository/keeping-your-actions-up-to-date-with-github-dependabot) and figured it wouldn't hurt to have it.